ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py

```diff
@@ -172,3 +172,36 @@ def test_sparse_categorical_crossentropy(
         y_pred=y_pred,
         from_logits=from_logits,
     )
+
+
+# mean_absolute_error
+@handle_cmd_line_args
+@given(
+    y_true=st.lists(st.floats(min_value=0., max_value=4.), min_size=1, max_size=1),
+    dtype_y_pred=helpers.dtype_and_values(
+        available_dtypes=ivy_tf.valid_float_dtypes,

```

Comment:
why did you use ivy_tf here instead of ivy.current_backend()?

Reply:
I was testing the code, and in Pycharm it passes all the tests, I'm not sure what's wrong here